doc/guides/modules.rst

```diff
@@ -22,14 +22,58 @@ should not contain code that is written exclusively for Zephyr.
      should not be submitted as a Zephyr module; instead, it should be
```

Comment:
typo in commit msg s/paragrapg/paragraph

Reply:
appreciate early feedback :)

Reply:
I understand that with the existing modules (e.g. mcumg) we would need to move the zephyr code out of them into the zephyr source tree?
Or you mean that modules that are consist of 100% zephyr should be part of zephyr rather than modules?